acceptance/setup/common/00_EnvSetup.rb

```diff
@@ -46,5 +46,11 @@
   when /solaris/
     step "#{host} Install json from rubygems"
     on host, 'gem install json'
+
+    step "#{host} Install deep_merge from rubygems"
+    on host, 'gem install deep_merge'
+  else
+    step "#{host} Install deep_merge from rubygems"
+    on host, 'gem install deep_merge'
```

Comment:
@MikaelSmith explained things to me. He's going to update the aio/options.rb file so that this common setup step is not executed during `aio` builds. So I'm :+1: